chart/k8gb/crd/dns-endpoint-crd-manifest.yaml

```diff
@@ -1,4 +1,3 @@
-{{- if .Values.k8gb.deployCrds }}
 ---
```

Comment:
does `---` needs to be here ? When brackets were used my controller-gen move it unintentionally.

Reply:
I had a mistake in my previous version of the PR.. it was appending the crds for k8gb to that file, now it's only `>` instead of `>>`.

`---` at the beginning is what `controller-gen crd:crdVersions=v1 paths="./..." output:crd:stdout` actually produces, I am not sure what you mean by "controllergen can move it" (perhaps it was the issue with append instead of rewrite logic?)

Reply:
@jkremser , thx for clarification.